### DISCUSSION TITLE
Problem of Open Source License

### DISCUSSION THREAD

**Discussion by linhanyu** [eclipse-theia/theia#10358] [Q&A]
I'm prepared to open source my project, which used theia to build a electron application. (Use Apache license). I had clone @theia/application-manager and @theia/electron two repo , save in third_party directory, and modify their package.json to support electron13 and node 14 to support M1 devices.
I noticed that theia use EPL2.0 nad GPLv2.0  with classpath exception. I'm not sure can I change package.json in my project, and did it violate the license for downstream project if I used Apache License.  

According to #9936.
I'm not sure is that ok for downstream Open Source project to update electron by edit package.json

**Comment by vince-fugnitto** [comment]
@linhanyu thank you for the discussion :+1:

I'm not a lawyer, but I believe you are unable to change the license in a downstream extension especially since you are essentially forking `@theia/application-manager` and `@theia/electron`.

We use `EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0` as a license and both of them state a condition where they must use **same license** for modifications:

> Modifications must be released under the same license when distributing the licensed material. In some cases a similar or related license may be used.

cc @marcdumais-work @brianking @tsmaeder
